Runtime/Game/Platforms/PlatformManager.cs

```diff
@@ -1,17 +1,106 @@
+        
+        private static readonly string[] PlatformStrings = new[]
+        {
+            "", // None
+            "guest", // Guest
+            "white_label_login", // WhiteLabel
+            "steam", // Steam
+            "psn", // PSN
+            "xbox_one", // XboxOne
+            "nintendo_switch", // NintendoSwitch
+            "amazon_luna", // AmazonLuna
+            "apple_sign_in" // AppleSignIn
+        };
+        private static readonly string[] PlatformFriendlyStrings = new[]
+        {
+            "None", // None
+            "Guest", // Guest
+            "White Label", // WhiteLabel
+            "Steam", // Steam
+            "Playstation Network", // PSN
+            "Xbox One", // XboxOne
+            "Nintendo Switch", // NintendoSwitch
+            "Amazon Luna", // AmazonLuna
+            "Apple Sign In" // AppleSignIn
+        };
+
+        private static Platforms currentPlatform;
+        private static string currentPlatformString;
+        private static string currentPlatformFriendlyString;
+
+        public override string ToString()
+        {
+            return currentPlatformString;
+        }
+
+        public static Platforms Get()
+        {
+            return currentPlatform;
+        }
+
+        public static string GetString()
+        {
+            return currentPlatformString;
+        }
+
+        public static string GetFriendlyString()
+        {
+            return currentPlatformFriendlyString;
+        }
+
+        public static void Set(Platforms platform)
+        {
+            currentPlatform = platform;
+            currentPlatformString = PlatformStrings[(int)currentPlatform];
+            currentPlatformFriendlyString = PlatformFriendlyStrings[(int)currentPlatform];
+        }
+
+        public static void Reset()
+        {
+            currentPlatform = Platforms.None;
+            currentPlatformString = PlatformStrings[(int)currentPlatform];
+            currentPlatformFriendlyString = PlatformFriendlyStrings[(int)currentPlatform];
+        }
+
+        // TODO: Deprecated, remove in version 1.2.0
+        public static void Set(LootLockerConfig.platformType platform)
+        {
+            switch (platform)
+            {
+                case LootLockerConfig.platformType.Android:
+                    Set(Platforms.Guest);
+                    break;
+                case LootLockerConfig.platformType.iOS:
+                    Set(Platforms.AppleSignIn);
+                    break;
+                case LootLockerConfig.platformType.Steam:
+                    Set(Platforms.Steam);
+                    break;
+                case LootLockerConfig.platformType.PlayStationNetwork:
+                    Set(Platforms.PlayStationNetwork);
+                    break;
+                case LootLockerConfig.platformType.Unused:
+                default:
+                    Set(Platforms.None);
+                    break;
+            }
+        }
```

Comment:
Feels unnecessary to have enum Platforms, PlatformStrings and PlatformFriendlyStrings for essentially the same information. You can get away with just the enum and a string, and still get the firendly name with some regex.
Example;
```
public void Set(Platforms platform)
    {
        currentPlatform = platform;
        currentPlatformString = Enum.GetName(typeof(Platforms), (int)currentPlatform);
        // Replace underscore with space
        currentPlatformString = Regex.Replace(currentPlatformString, "_", " ");
        // Make the first character uppercase
        currentPlatformString= Regex.Replace(currentPlatformString, @"\b[a-z]", match => match.Value.ToUpper());
        Debug.Log(currentPlatformString); // example: amazon_luna becomes Amazon Luna
    }
```
But the solution you have now works of course, just thought it felt unnecessary to essentially repeat the same information in 3 places with just slightly different formatting.

Reply:
Not really. Of course most people will know what PSN is, but this let's us have any arbitrary descriptive naming for a platform for logging even if the platform names are acronyms which a Regex doesn't. Also, a Regex is more difficult so the likelihood of someone adding a platform and then touching up the regex and messing it up is not low.

Reply:
Agree with the regex. 
Still don't think we need 3 variables for it.
Can we remove the "platform"-array?
Keep the friendly_platform, but remove the "platform" and use Enum.GetName() when returning that?

Reply:
I mean, sure, technically that is feasible. I'm unsure if I like it though. The enum names and the friendly strings are customer-facing and intended to be understandable for our users. The Platform strings one is strictly a representation of what's in the database in the backend and needs to match exactly, thus we are at the whim of what they name the platforms.

Reply:
Let's leave it as it is for now, and we can re-evaluate when we launch a new platform if it is something that we think is very annoying or not. 👍